book/src/installation_and_usage.md

```diff
@@ -29,7 +32,7 @@ rustup component add clippy
 ```
 If it says that it can't find the `clippy` component, please run `rustup self update`.
 
-#### Step 3: Run Clippy
+### Step 3: Run Clippy
```

Comment:
Since Clippy is most likely pre-installed I would split up installation and usage. Most people will only be interested in the usage part.

---

EDIT:

> Actually I think I've changed my mind on this for now. It's currently pretty short and still covers the basics. I suggest we leave it as is, and then consider a more in depth standalone usage page in the next round with much more detail, including everything currently there plus more in depth command line and in code usages.

Just saw your comment here. This makes sense, we can make the split in a follow up PR. ~~Could you track open tasks (not addressed in this PR) in the PR body please?~~ I put it into the PR body.